src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -274,6 +279,13 @@ void icm426xxGyroInit(gyroDev_t *gyro)
 
     spiWriteReg(dev, ICM426XX_RA_INT_CONFIG1, intConfig1Value);
 
+    // Disable AFSR to prevent stalls in gyro output
+    uint8_t intfConfig1Value = spiReadRegMsk(dev, ICM426XX_INTF_CONFIG1);
```

Comment:
Add comment about ICM426XX_INTF_CONFIG1 location in user bank 0
Also, is is really necessary use 3 lines for mask-and-set operation, which is idiomatic in low level programming ?
```
spiWriteReg(dev, ICM426XX_INTF_CONFIG1, (intfConfig1 & ~ICM426XX_INTF_CONFIG1_AFSR_MASK) | ICM426XX_INTF_CONFIG1_AFSR_DISABLE);

Reply:
Agreed, but I simply followed the style of the adjacent code. I tend to try to follow the style of any code being maintained to avoid a confusing mix. Happy to make the obvious enhancement in which case I’d do the same for the other initialisation.

Reply:
I know. This is just minor case, and maybe only my personal preference. 
The code style will slowly spread as new drivers are written based on old ones. Maybe mixing both is not that dangerous - with little luck, someone will use better one when creating new driver.